lib/process-services-cloud/src/lib/form/components/form-cloud.component.ts

```diff
@@ -421,6 +422,7 @@ export class FormCloudComponent extends FormBaseComponent implements OnChanges,
             this.displayModeOn.emit(this.displayModeService.findConfiguration(this.displayMode, this.displayModeConfigurations));
         }
 
+        this.changeDetector.detectChanges();
```

Comment:
Why do you need to call the detectChanges() here?

Reply:
currently, in case where some field is empty and required at the initial state, we need to update the form state. Because its initial state and no action was performed, form doesn't now that empty required field is invalid, so we need to do it manually at the widgets level (in the ngOnInit). And because we need to do it manually, we change the state after it was checked and we get the Expression has been change after it was checked error. It should be solved once we move the whole form to be reactive instead of manipulate it manually as we currently do.